frontend/public/components/monitoring.tsx

```diff
@@ -1105,9 +1101,10 @@ const QueryBrowserPage = withFallback(() => {
 
   const onQueryChange = (e, i) => updateQuery(i, {text: e.target.value});
 
-  const toggleEnabled = i => _.get(queries, [i, 'enabled'])
-    ? updateQuery(i, {enabled: false, expanded: false, query: ''})
-    : updateQuery(i, {enabled: true, expanded: true, query: _.get(queries, [i, 'text'])});
+  const toggleEnabled = i => {
```

Comment:
```suggestion
  const toggleEnabled = (i: number) => {
```

Reply:
I've added some type definitions like this one in https://github.com/openshift/console/pull/1817